package.json

```diff
@@ -39,18 +45,18 @@
     "test": "graph test"
   },
   "dependencies": {
-    "@graphprotocol/graph-cli": "0.33.1",
-    "@graphprotocol/graph-ts": "0.27.0",
+    "@graphprotocol/graph-cli": "0.37.0",
```

Comment:
Sadly, we can't go to the latest version because compilation fails and it's impossible to debug.